src/app/models/feedback.py

```diff
@@ -0,0 +1,63 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.sql.expression import select
+from sqlalchemy.orm import aliased
+from sqlalchemy.engine import Result
+from src.core.database.models.feedback import DataModelPredictionFeedback as dbFeedback
+from src.core.database.models.player import Player as dbPlayer
+from src.app.views.response.feedback import PredictionFeedbackResponse
+import logging
+
+
+class AppModelFeedback:
+    def __init__(self, session: AsyncSession) -> None:
+        self.session = session
+        self.logger = logging.getLogger("AppModelFeedback")
+
+    async def get_feedback_responses(self, player_names: list[str]):
+        async with self.session:
+            print(f"Player names: {player_names}")
+            fv: dbFeedback = aliased(dbFeedback, name="feedback_voter")
+            fs: dbFeedback = aliased(dbFeedback, name="feedback_subject")
+            pn: dbPlayer = aliased(dbPlayer, name="player_name")
```

Comment:
this could be a bit more verbose, fb_voter, fb_subject, pl_name

Reply:
resolved with [e260e5f](https://github.com/Bot-detector/public-api/pull/25/commits/e260e5f534fa87252579a96a232b701f1f35906d)